app/main.py

```diff
@@ -1,2 +1,86 @@
+from typing import Any
+
+import copy
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.hash_table: list = [None] * 8
+        self.len_hash_tabel = len(self.hash_table)
+
+    def __len__(self) -> int:
+        return self.length
+
+    def get_cell_index(self, key: Any) -> int:
+        return hash(key) % self.len_hash_tabel
+
+    def resize_hash_tabel(self) -> None:
+        old_tabel = copy.deepcopy(self.hash_table)
+        self.hash_table = self.len_hash_tabel * [None] * 2
```

Comment:
Incorrect initialization of the new hash table. The expression `self.len_hash_tabel * [None] * 2` will not double the size of the table as intended. It should be `[None] * (self.len_hash_tabel * 2)`.